_health-care/_js/_components/mothers_maiden_name.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import _ from 'lodash';
+
+class MothersMaidenName extends React.Component {
+  constructor() {
+    super();
+    this.state = {name: ''};
+    this.handleNameChange = this.handleNameChange.bind(this);
+  }
+
+  handleNameChange(event) {
+    this.setState({name: event.target.value});
+  }
+
+  render() {
+    return (
+      <div>
+        <div className="usa-input-grid usa-input-grid-large">
+          <label htmlFor="veteran_mothers_maiden_name">Mother’s Maiden Name</label>
+          <input type="text" name="veteran[mothers_maiden_name]"
```

Comment:
For required, I assume that it's going to be a meta component and that each component is going to need some sort of configurability.  Something like

```
render() {
   <RequiredComponent target={MothersMaidenName} target_args={state: this.veteran.mothers_maiden_name} />
   <DateInput max_age=50 />
}
```